apps/api/src/app/plugins/bffCache.ts

```diff
@@ -17,6 +18,8 @@ export const bffCache: FastifyPluginCallback<BffCacheOptions> = (fastify, opts,
     }
 
     let key = getKey(request)
+    // Remove it so we can cache it properly
+    request.headers['accept-encoding'] = undefined;
```

Comment:
This is needed since I don't know how to handle decompressing and re-compressing the reply afterwards.